packages/plugins/response-cache/src/plugin.ts

```diff
@@ -331,7 +331,8 @@ export function useResponseCache<PluginContext extends Record<string, any> = {}>
           [MapperKind.COMPOSITE_TYPE]: type => {
             const cacheControlAnnotations = getDirective(schema, type, 'cacheControl');
             cacheControlAnnotations?.forEach(cacheControl => {
-              const ttl = cacheControl.maxAge * 1000;
+              const ttl = cacheControl.maxAge != null ? cacheControl.maxAge * 1000 : null;
+
               if (ttl != null) {
                 ttlPerType[type.name] = ttl;
               }
```

Comment:
Do we really need the local variable here ?
```suggestion
              if (cacheControl.maxAge != null) {
                ttlPerType[type.name] = cacheControl.maxAge * 1000;
              }
```

Reply:
Fixed